ceph-pull-requests/config/definitions/ceph-pull-requests.yml

```diff
@@ -28,7 +28,8 @@
           allow-whitelist-orgs-as-admins: true
           org-list:
             - ceph
-          trigger-phrase: ''
+          trigger-phrase: ''    
+          skip-build-phrase: 'qa tests .* skip build'
```

Comment:
I think this is not a regex or a list of regexes/phrases, but just a single string that if present in the title, will skip the build. So maybe something like "NO-TEST" or "SKIP-TEST" ? It will also need to be added to the other "make check" job for arm64